test/OpenTelemetry.Extensions.Docker.Tests/Resources/TestDockerResourceDetector.cs

```diff
@@ -0,0 +1,112 @@
+// <copyright file="TestDockerResourceDetector.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.IO;
+using System.Linq;
+using OpenTelemetry.Extensions.Docker.Resources;
+using OpenTelemetry.Resources;
+using Xunit;
+
+namespace OpenTelemetry.Extensions.Docker.Tests.Resources
+{
+    public class TestDockerResourceDetector
```

Comment:
Rename this to `DockerResourceDetectorTests`. That's the convention followed in the contrib repo.